java/scene-property-expressions/README.md

```diff
@@ -2,53 +2,35 @@
 
 Update the orientation of a graphic using expressions based on its attributes.
 
-![Scene property expressions App](scene-property-expressions.png)
+![Image of scene property expressions](scene-property-expressions.png)
 
 ## Use case
 
-Instead of reading the attribute and changing the rotation on the symbol
-for a single graphic (a manual CPU operation), you can bind the rotation
-to an expression that applies to the whole overlay (an automatic GPU
-operation). This usually results in a noticeable performance boost
-(smooth rotations).
+Instead of reading the attribute and changing the rotation on the symbol for a single graphic (a manual CPU operation), you can bind the rotation to an expression that applies to the whole overlay (an automatic GPU operation). This usually results in a noticeable performance boost (smooth rotations).
 
 ## How to use the sample
 
-Use the seek bars to adjust the heading and pitch properties for the
-graphic.
+Adjust the heading and pitch sliders to rotate the cone.
 
 ## How it works
 
-Simple renderers can dynamically update the positions of graphics using
-an expression. The expression relates a renderer property to one of the
-graphic's attribute keys.
-
 1. Create a new `GraphicsOverlay`. 
 2. Create a new `SimpleRenderer` and set its scene properties.
```

Comment:
This step seems redundant to me as it looks like the only properties that are set are the expressions, which are included in the next step